application/src/main/data/json/system/widget_bundles/edge_widgets.json

```diff
@@ -2,12 +2,15 @@
+      "description": "Displays list of the assigned to edge entities and customer's title.",
```

Comment:
Let's change it to or something similar:
'Overview of entities related to ThingsBoard Edge.'